src/util-profiling.c

```diff
@@ -118,7 +118,7 @@ thread_local int profiling_rules_entered = 0;
 
 void SCProfilingDumpPacketStats(void);
 const char * PacketProfileDetectIdToString(PacketProfileDetectId id);
-const char * PacketProfileLoggertIdToString(LoggerId id);
+const char *PacketProfileLoggerIdToString(LoggerId id);
```

Comment:
I blame the formatter for the whitespace change here (my tooling doesn't touch whitespace at all)